Makefile

```diff
@@ -171,8 +173,12 @@ build-all-push-all-test-e2e: all-images push-all-images test-e2e ## Builds and p
 # flag), and ginkgo's own timeout.
 .PHONY: test-e2e
 test-e2e: ## Run the end-to-end tests. When testing local code, container images should be built beforehand (or use target build-images-test-e2e).
+ifdef GINKGO_FOCUS
+	go run github.com/onsi/ginkgo/v2/ginkgo -v -focus="$(GINKGO_FOCUS)" test/e2e
```

Comment:
You are probably aware of it, but just in case: you can also simply change a specific `Describe`, `DescribeTable` or `It` to `FDescribe`, `FDescribeTable` or `FIt` temporarily to focus on a particular suite or test case. I find this often more convenient compared to providing -focus. Similar with `XDescribe`, `XIt` to exclude test cases.

Either way, having support for -focus in the Makefile is definitely a good addition. 👍